security/security_profiles_operator/spo-release-notes.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+[id="spo-release-notes"]
+= Security Profiles Operator release notes
+include::_attributes/common-attributes.adoc[]
+:context: spo-release-notes
+
+toc::[]
+
+The Security Profiles Operator provides a way to define secure computing (https://kubernetes.io/docs/tutorials/security/seccomp/[seccomp]) and SELinux profiles as Custom Resources, synchronizing profiles to every node in a given namespace.
+
+These release notes track the development of the Security Profiles Operator in {product-title}.
+
+For an overview of the Security Profiles Operator, see xref:../security_profiles_operator/spo-overview.adoc#[Security Profiles Operator Overview].
+
+== Security Profiles Operator x.x.x
```

Comment:
If you want to fill in the numbers, we are going to release 0.5.0 + selected patches from upstream